spec/requests/properties_spec.rb

```diff
@@ -187,123 +187,194 @@
+                  "endDate" => nil
+                },
+                {
+                  "type" => "VA",
+                  "comments" => "Verbal Abuse or Threat of",
+                  "startDate" => "2018-04-13",
+                  "endDate" => nil
                 }
               ]
             },
-          status: 200
-        )
-
-        get("/api/v2/properties/100023022310", headers: headers)
+            "tenure" => {
+              "typeCode" => "SEC",
+              "typeDescription" => "Secure"
+            }
+          )
+        end
       end
 
-      it "returns a JSON representation from a single property" do
-        parsed_response = JSON.parse(response.body)
-
-        expect(parsed_response).to eq(
-          "property" => {
-            "propertyReference" => "100023022310",
-            "address" => {
-              "shortAddress" => "1 Example Road",
-              "postalCode" => "A1 1AA",
-              "addressLine" => "1 Example Road",
-              "streetSuffix" => ""
-            },
-            "hierarchyType" => {
-              "levelCode" => "1",
-              "subTypeCode" => "DWE",
-              "subTypeDescription" => "Dwelling"
-            }
-          },
-          "alerts" => {
-            "locationAlert" => [
+      context "with no tenancy information details" do
+        before do
+          stub_property_request(
+            reference: "100012345678",
+            response_body:
               {
-                "type" => "DIS",
-                "comments" => "Property Under Disrepair",
-                "startDate" => "2011-02-16",
-                "endDate" => nil
-              }
-            ],
-            "personAlert" => [
+                propRef: "100012345678",
+                address1: "1 Example Road",
+                postCode: "A1 1AA",
+                levelCode: "1",
+                subtypCode: "DWE"
+              },
+            status: 200
+          )
+
+          stub_alerts_property_request(
+            reference: "100012345678",
+            response_body:
               {
-                "type" => "CV",
-                "comments" => "No Lone Visits",
-                "startDate" => "2018-04-13",
-                "endDate" => nil
+                "propertyReference": "100012345678",
+                "alerts": []
               },
+            status: 200
+          )
+
+          stub_tenancy_information_property_request(
+            reference: "100012345678",
+            response_body:
               {
-                "type" => "VA",
-                "comments" => "Verbal Abuse or Threat of",
-                "startDate" => "2018-04-13",
-                "endDate" => nil
+                "tenancies": []
+              },
+            status: 200
+          )
+
+          get("/api/v2/properties/100012345678", headers: headers)
+        end
+
+        it "returns a JSON representation from a single property with tenure=nil and personAlert=[]" do
```

Comment:
I suppose `locationAlert=[]` as well so you could just say no alerts . It's probably better to have tests as human readable so I would also say no tenure details instead of `tenure=nil` too